vbox/Dockerfile

```diff
@@ -89,9 +89,11 @@ VOLUME ["/cuckoo/conf"]
 
 EXPOSE 1337 31337
 
-ADD --chown=cuckoo:cuckoo https://raw.githubusercontent.com/$CUCKOO_FORK_REPO/cuckoo/machinery/virtualbox_websrv.py $CUCKOO_PACKAGE_PATH/machinery/virtualbox_websrv.py
-ADD --chown=cuckoo:cuckoo https://raw.githubusercontent.com/$CUCKOO_FORK_REPO/cuckoo/common/config.py $CUCKOO_PACKAGE_PATH/common/config.py
+ADD https://raw.githubusercontent.com/$CUCKOO_FORK_REPO/cuckoo/machinery/virtualbox_websrv.py $CUCKOO_PACKAGE_PATH/machinery/virtualbox_websrv.py
+ADD https://raw.githubusercontent.com/$CUCKOO_FORK_REPO/cuckoo/common/config.py $CUCKOO_PACKAGE_PATH/common/config.py
 
-USER cuckoo
+RUN apk add shadow \
```

Comment:
I'm not sure this is necessary or not, but a apk add --no-cache will also do an update and install, but then will clean up the apk install cache decreasing the image size ever so slightly.